### ISSUE TITLE
URGENT! Error: undefined: urlstruct.OpEq with pg/v9

### ISSUE THREAD

**Issue by frederikhors** [go-pg/pg#1703]
Using go.mod:

```go.mod
...
github.com/go-pg/pg/v9 v9.1.7
github.com/go-pg/urlstruct v0.5.2 // indirect
...
```

I'm having this error today:

```console
C:\projects\go\pkg\mod\github.com\go-pg\pg\v9@v9.1.7\orm\struct_filter.go:12:2: undefined: urlstruct.OpEq
C:\projects\go\pkg\mod\github.com\go-pg\pg\v9@v9.1.7\orm\struct_filter.go:13:2: undefined: urlstruct.OpNotEq
C:\projects\go\pkg\mod\github.com\go-pg\pg\v9@v9.1.7\orm\struct_filter.go:14:2: undefined: urlstruct.OpLT
C:\projects\go\pkg\mod\github.com\go-pg\pg\v9@v9.1.7\orm\struct_filter.go:15:2: undefined: urlstruct.OpLTE
C:\projects\go\pkg\mod\github.com\go-pg\pg\v9@v9.1.7\orm\struct_filter.go:16:2: undefined: urlstruct.OpGT
C:\projects\go\pkg\mod\github.com\go-pg\pg\v9@v9.1.7\orm\struct_filter.go:17:2: undefined: urlstruct.OpGTE
C:\projects\go\pkg\mod\github.com\go-pg\pg\v9@v9.1.7\orm\struct_filter.go:18:2: undefined: urlstruct.OpIEq
C:\projects\go\pkg\mod\github.com\go-pg\pg\v9@v9.1.7\orm\struct_filter.go:19:2: undefined: urlstruct.OpMatch
C:\projects\go\pkg\mod\github.com\go-pg\pg\v9@v9.1.7\orm\struct_filter.go:23:2: undefined: urlstruct.OpEq
C:\projects\go\pkg\mod\github.com\go-pg\pg\v9@v9.1.7\orm\struct_filter.go:27:29: undefined: urlstruct.OpCode
C:\projects\go\pkg\mod\github.com\go-pg\pg\v9@v9.1.7\orm\struct_filter.go:23:2: too many errors
```

**Comment by vmihailenco** [comment]
You should use `github.com/go-pg/urlstruct v0.4.0`. How have you ended up with "v0.5.2"?

**Comment by frederikhors** [comment]
`github.com/go-pg/urlstruct v0.5.2` is `// indirect`.

I get it by using `go get -u` having `go-pg/pg/v9` in my `go.mod`.

**Comment by hazcod** [comment]
Same issue for me with `github.com/go-pg/pg/v9 v9.1.7`

**Comment by vmihailenco** [comment]
I've retagged v0.5.x as v1.0.0 - hopefully go mod will not auto upgrade any more.

**Comment by frederikhors** [comment]
It still gets `github.com/go-pg/urlstruct v0.5.2 // indirect`.

Same problem.

**The issue is NOT fixed.**

**Comment by hazcod** [comment]
I guess you need to bump the version  in `pg` @vmihailenco

**Comment by frederikhors** [comment]
Now the same problem but with `github.com/go-pg/urlstruct v1.0.0 // indirect` in `go.mod`.

**Comment by frederikhors** [comment]
@hazcod can you please explain it better to me? What is there to do to fix this?

**Comment by hazcod** [comment]
Not sure, can't see where the library is used

**Comment by vmihailenco** [comment]
I've released v9.2.0 which does not have urlstruct dependency.

**Comment by frederikhors** [comment]
It works now, thanks.
